readme.md

```diff
@@ -7,6 +7,35 @@ $ npm -g install gimme-stat
 $ cd /home/your_git_project_name
 $ gimme-stat
 ```
+# Usage
+Gimme-that can be used thru terminal and as required module. Way you like to use it will be detected automatically.
```

Comment:
I think we can remove the line. 
"Easiest way to use it is to call module with default parameters." is enough clear.